Spinel Changes

darconeous opened the pull request:
This pull request contains various minor changes to help with readability.

jhwoodyatt commented:
LGTM.

jwhui commented on doc/spinel-protocol-src/draft-rquattle-spinel-basis.md.in:
```diff
@@ -50,23 +49,13 @@
 # Introduction #
 
-Spinel is designed to enable the interoperation over simple serial connections between general purpose device operating systems (OS) and network co-processors (NCP) for the purpose of controlling and managing their IPv6 network interfaces, achieving the following goals:
+Spinel is host-controller protocol designed to enable interoperation over simple serial connections between general purpose device operating systems (OS) and network co-processors (NCP) for the purpose of controlling and managing their IPv6 network interfaces, achieving the following goals:
```
nit: "is host-controller" -> "is **a** host-controller"?

jwhui commented on doc/spinel-protocol-src/draft-rquattle-spinel-unified.md.in:
```diff
@@ -51,42 +50,22 @@
-This Network Control Processor (NCP) protocol was designed to enable a
-host device to communicate with and manage a NCP while also achieving
-the following goals:
+Spinel is host-controller protocol designed to enable interoperation over simple serial connections between general purpose device operating systems (OS) and network co-processors (NCP) for the purpose of controlling and managing their IPv6 network interfaces, achieving the following goals:
```
nit: "is host-controller" -> "is **a** host-controller"

jwhui commented on doc/spinel-protocol-src/spinel-prop-overview.md:
```diff
@@ -0,0 +1,69 @@
+
+Single-value properties are properties that have a simple representation of a single value. Examples would be:
+
+*   Current radio channel (Represented as a unsigned 8-bit integer)
```
nit: "as a unsigned" -> "as **an** unsigned"

jwhui commented on doc/spinel-protocol-src/spinel-prop-overview.md:
```diff
@@ -0,0 +1,69 @@
+
+*   Current radio channel (Represented as a unsigned 8-bit integer)
+*   Network name (Represented as a UTF-8 encoded string)
+*   802\.15.4 PAN ID (Represented as a unsigned 16-bit integer)
```
nit: "as a unsigned" -> "as **an** unsigned"

darconeous commented:
In the future, I think minor grammar errors should be addressed in follow up commits until we get this moved over into a new repository. I would have addressed these issues directly in the GitHub web interface, but I can't pick the email address used in the attribution and that causes googlebot to wig out.
